packages/office-ui-fabric-react/src/components/ResizeGroup/examples/ResizeGroup.OverflowSet.Example.tsx

```diff
@@ -0,0 +1,80 @@
+/* tslint:disable:no-unused-variable */
```

Comment:
we should be able to do this in tslint.json I think.. `"react", true` as value for that setting